draft-ietf-jsonpath-base.md

```diff
@@ -1142,30 +1125,6 @@ Some examples:
 | `{"col":"red"}` | `$[?@ in ['red','green','blue']]` | `["red"]` | Containment |
 | `{"a":{"b":{5},c:0}}` | `$[?@.b==5 && !@.c]` | `[{"b":{5},c:0}]` | Existence  |
 
-
-# Expression Language
-
-> Task (T2): Separate out expression language.  For now, this section
-> is a repository for ABNF taken from {{-json}}.  This needs to be
-> deduplicated with definitions above.
-
-~~~~ abnf
```

Comment:
We need to keep `number`, `true`, `false`, `null`.
(We also need to define slice-index.)

Reply:
> We need to keep number, true, false, null.

Oops. I reintroduced them.

> (We also need to define slice-index.)

Noted. Out of scope for this PR.